factcast-store-pgsql/src/main/java/org/factcast/store/pgsql/registry/validation/schema/store/PgSchemaStoreImpl.java

```diff
@@ -41,6 +41,7 @@
     // note that SpringLiquibase needs to be injected in order to make sure it
     // is initialized before we're intentionally not using @DependsOn here,
     // as a change of the beanname within liquibase would break our code
+    @SuppressWarnings("unused")
     private final SpringLiquibase unused;
```

Comment:
Maybe we should remove this constructor param from here and move the comment to the configuration class?

Reply:
:+1: